resources/public/api/conf/1.0/reliefInvestments_retrieve.raml

```diff
@@ -0,0 +1,27 @@
+is:
+  - headers.acceptJson
+  - headers.contentTypeJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.formatTaxYear
+  - errors.ruleTaxYearRangeInvalid
```

Comment:
I tried running the RAML locally and it errors saying there is no reference for this. Does this error exist in the errors.raml?